dist/gmusic.js

```diff
@@ -195,6 +204,38 @@ proto.playback = {
     this.playback._sliderEl.dispatchEvent(evt);
   },
 
+  getPlaybackState: function () {
+    // Play/Pause element states:
+    //   PLAYING: {__data__: {icon: 'av:pause-circle-filled'}, disabled: false}
+    //   PAUSED: {__data__: {icon: 'av:sj:pause-circle-fill'}, disabled: false}
+    //   STOPPED: {__data__: {icon: 'av:sj:play-circle-fill'}, disabled: true}
+    if (!this.playback._playPauseEl.disabled) {
+      if (this.playback._playPauseEl.__data__.icon === 'av:pause-circle-filled') {
+        return GMusic.Playback.PLAYING;
+      } else {
+        return GMusic.Playback.PAUSED;
+      }
+    } else {
+      return GMusic.Playback.STOPPED;
+    }
+  },
+
+  getSongInfo: function () {
+    var songInfo = {};
+    songInfo.title = this.doc.getElementById(SELECTORS.info.titleId).textContent || 'Unknown';
+    songInfo.artist = this.doc.getElementById(SELECTORS.info.artistId).textContent || 'Unknown';
+    songInfo.album = this.doc.querySelector(SELECTORS.info.albumSelector).textContent || 'Unknown';
+    songInfo.art = this.doc.getElementById(SELECTORS.info.albumArtId) || null;
+    songInfo.duration = this.doc.getElementById(SELECTORS.playback.sliderId).max;
```

Comment:
It looks like a comment about putting this into an object was buried. Can we still do that?

```
var songInfo = {
  title: ...
};
```